src/components/layout/dashboard/DashboardContentContainer.tsx

```diff
@@ -115,16 +118,17 @@ const DashboardContentContainer: React.FC<Props> = ({
             </Paper>
           )}
           <Box
+            key='content'
+            component='main'
             sx={{
               pt: 2,
               pb: 8,
               px: { xs: 1, sm: 3, lg: 4 },
               maxWidth: `${maxPageWidth}px`,
+              ...contentSx,
             }}
           >
-            <Box key='content' component='main'>
-              <SentryErrorBoundary>{children}</SentryErrorBoundary>
-            </Box>
+            <SentryErrorBoundary>{children}</SentryErrorBoundary>
```

Comment:
This feels like it could be a big change, there might be some styling somewhere that expects two divs. Did you check to see if there is any impact on the dashboards?

Reply:
I did check the dashboards, both mobile and desktop, and they look okay. However, of course I'm not super confident that there isn't any styling anywhere that doesn't expect 2 divs here, so I'd be happy to revert this change and leave it be.

The reason to make the change in the first place was only code-stylistic, anyway -- I liked the idea of `DashboardContentContainer` accepting a `contentSx` prop that it applies to the div labelled 'content', whereas with the 2-Box layout it needs to accept an `sx` prop and pass it to the Box one level above 'content,' and I just wasn't sure what to name that sx prop, haha!

Let me know what you think, I could go either way.

Reply:
great! I think it's a nice clean-up, I think we should keep it.